source/usage-examples/command.txt

```diff
@@ -0,0 +1,48 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
```

Comment:
Although I know this resembles the Node usage example, I'm currently wondering whether it appropriately addresses the Jasmine persona.  If we re-order some of the information, I think we could improve the learning experience. E.g.

"You can run all raw database operations ... method. A raw database operation is a command you can execute directly on the MongoDB server CLI and includes administrative and diagnostic commands such as fetching server stats or initializing a replica set. Call the ``runCommand()`` method with a Bson command object on an instance of a database to run your raw database operation."

Reply:
Will try out a couple of different orders, see what sticks.